src/NUnit3TestAdapterInstallV2/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("NUnit3TestAdapterInstall")]
-[assembly: AssemblyCopyright("")]
+[assembly: AssemblyCopyright("Copyright © 2011-2016, NUnit Software")]
```

Comment:
Both the NUnit 3 and NUnit 2 adapters say that they are copyright Charlie Poole in some places and NUnit Software in other places. Let's not further complicate things by adding another thing we may have to change.

I understand you're taking this from the discussion we had about the test generator, but that was a special case because of the Microsoft involvement and I wasn't intending it to be a global decision. In fact, I don't think a global decision is possible wrt copyright - it has to reflect the actual facts in each case. I think we can resolve this after the release and I'll create an issue for it.